syz-cluster/kernel-disk/cron.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2025 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: kernel-repo-update
+spec:
+  schedule: "0 */6 * * *" # Update every 6 hours
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: Never
+          volumes:
+            - name: git-repo
+              persistentVolumeClaim:
+                claimName: base-kernel-repo-pv-claim
+          containers:
+            - name: git-updater-debug-job
+              image: alpine/git:latest
+              imagePullPolicy: IfNotPresent
+              volumeMounts:
+                - name: git-repo
+                  mountPath: /repo.git
+              resources:
+                requests:
+                  cpu: 4
+                  memory: 8G
+                limits:
```

Comment:
Is it needed to prevent throttling and OOMs?

Reply:
It is better to document then. I used different sources to understand what's going on here.

Reply:
I'd say these are very basic K8S/GKE configuration options, we don't really want to duplicate k8s documentation snippets across the code.

Once we start moving to GKE, we'd have to set resource limits for all k8s deployments and jobs.